modules/bridge/src/main/ts/ephox/bridge/core/MenuButton.ts

```diff
@@ -6,19 +6,28 @@ import { NestedMenuItemContents } from '../components/menu/NestedMenuItem';
 export type MenuButtonItemTypes = NestedMenuItemContents;
 export type SuccessCallback = (menu: string | MenuButtonItemTypes[]) => void;
 
+// NOTE: FetchContext is an object so that we can add information to it in the future.
+export interface FetchContext {
```

Comment:
This might need adding to the `PublicMenu.ts` and `Menu.ts` files otherwise we won't be able to use it. If doing that then we'll also need to rename this to something like `MenuButtonFetchContext` so we don't run into that issue you had with `PresetTypes` where it was specific to split toolbar buttons.

Reply:
As discussed, it's probably most similar to `SuccessCallback` and we aren't currently exporting that. So we'll leavei t out of the public types for now.